src/renderers/server-render.jsx

```diff
@@ -124,6 +118,15 @@ const getRouterRenderer = (
 	};
 };
 
+const makeRenderer$ = (config: Object) => makeRenderer(
+		config.routes,
+		config.reducer,
+		config.clientFilename,
+		config.assetPublicPath,
+		config.middleware,
+		config.baseUrl
+	);
```

Comment:
it would be helpful to define a `type` for this `config` - there are lots of `config` objects floating around server code.

```js
type ServerRenderConfig = {
  routes: Array<Object>,  // we'll define a `Route` type at some point to make this `Array<Route>`
  reducer: (state: ?Object, action: FluxStandardAction) => Object,
  ...
}
```